conversation_test.go

```diff
@@ -411,6 +413,28 @@ func TestGetConversationInfo(t *testing.T) {
 	}
 }
 
+func TestGetConversationInfoNilError(t *testing.T) {
+	http.HandleFunc("/conversations.info", okChannelJsonHandler)
+	once.Do(startServer)
+	api := New("testing-token", OptionAPIURL("http://"+serverAddr+"/"))
+	_, err := api.GetConversationInfo(nil)
+	if err == nil {
+		t.Errorf("Unexpected pass where there should have been nil input error")
+		return
+	}
+}
+
+func TestGetConversationInfoMissingChannelError(t *testing.T) {
+	http.HandleFunc("/conversations.info", okChannelJsonHandler)
```

Comment:
Some tests are failed in CI.
https://github.com/slack-go/slack/runs/6276344051?check_suite_focus=true

Reply:
Argh, this is due to them all sharing the http handler function. I've combined them to resolve this